test/utils/getUrl.js

```diff
@@ -31,3 +32,10 @@ getUrl.noAuth = function getUrlNoAuth(config, app) {
   });
   return getUrl(config, app);
 };
+
+getUrl.baseUrl = function getBaseUrl(config) {
+  config = _.pick(config, function (val, param) {
+    return param !== 'auth' & param !== 'pathname' & param !== 'hash';
```

Comment:
I think I should change this to include (instead of exclude) protocol, hostname, port.  But I'm not sure I'm doing this the best way.